region/maridia/inner-pink/Botwoon Hallway.json

```
@@ -405,6 +405,110 @@
         "This is for the much harder version, without Gravity and Bombs."
       ]
     },
+    {
+      "link": [1, 2],
+      "name": "10 Power Bomb Crystal Flash Clip with Bombs (Left to Right)",
+      "requires": [
+        "Bombs",
+        "Gravity",
+        "h_10PowerBombCrystalFlash",
+        {"resetRoom": {"nodes": [1]}}
+      ],
+      "flashSuitChecked": true,
+      "note": [
+        "Lure a Puyo close to the crumble blocks and time the first bomb with the Puyo's jump so that the bomb will kill the Puyo."
+      ]
+    },
+    {
+      "link": [1, 2],
+      "name": "Suitless 10 Power Bomb Crystal Flash Clip (Left to Right)",
+      "requires": [
+        "canBeVeryPatient",
+        "canSuitlessMaridia",
+        {"or": [
+            "HiJump",
+            "canSpringBallJumpMidAir",
+            {
+              "and": [
+                "h_underwaterCrouchJumpDownGrab",
+                "canCarefulJump"
+            ]}
+        ]},
```

```suggestion
       "h_underwaterCrouchJumpDownGrab",
```
CF eats your flash suit so there is no reason to complicate this.